onnxruntime/test/onnx/microbenchmark/layer_normalization.cc

```diff
@@ -111,9 +111,9 @@ static void BM_LayerNormalization(benchmark::State& state) {
   OrtMemoryInfo memory_info(onnxruntime::CPU, OrtAllocatorType::OrtArenaAllocator);
   AllocatorPtr alloc = std::make_shared<CPUAllocator>(memory_info);
   for (auto _ : state) {
-    auto status = layer_norm_impl.ComputeWithoutContext(x_data, x_shape, scale_data, scale_shape, bias_data, bias_shape,
-                                                        Y_data, mean_data, inv_std_dev_data, thread_pool.get(), axis,
-                                                        epsilon, simplified, alloc);
+    auto status = layer_norm_impl.ComputeWithoutContext(x_data, x_shape, scale_data, scale_shape.Size(), bias_data,
```

Comment:
need cast here: scale_shape.Size() is int64, it is casted to size_t, which might be int32 for x86.